std_msgs/String type
---
nav_msgs/OccupancyGrid map
